server/lib/sources/mock.js

```diff
@@ -156,6 +158,20 @@ let generateOne = account => {
     };
 };
 
+let generateRandomError = () => {
+    let errorTable = [];
+    // We diable the linter, because the error guard-for-in
+    // cannot be fixed
+
+    /* eslint-disable */
+    for (let error in errors) {
```

Comment:
tip: `for (let error of Object.keys(errors))` and you can remove the comment and the two eslint guards :+1: